pyrasterframes/src/main/python/docs/raster-write.pymd

```diff
@@ -85,6 +84,19 @@ If there are many _tile_ or projected raster columns in the DataFrame, the GeoTI
 os.remove(outfile)
 ```
 
+## Overview Rasters
+
+In cases where writing and reading to/from a GeoTIFF isn't convenient, RasterFrames provides the `rf_agg_overview_raster` aggregate function, where you can construct a single raster (rendered as a tile) downsampled from all or a subset of the dataframe. This allows you to effectively construct the same operations the GeoTIFF writer performs, but without the file I/O.
+
+Because a Dataframe may contain data with varying CRSs, and the rendered raster needs to have a single CRS, an "Area of Interest" (AOI) is required in a predetermined CRS. In the case of `rf_agg_reprojected_extent`, the AOI needs to be in commonly used ["web mercator"](https://en.wikipedia.org/wiki/Web_Mercator_projection) CRS. 
```

Comment:
the discussion about rf_agg_reprojected_extent is confusing because we have not explained what that is. Also nice here to link  to reference page on at least first mention of each function.